src/core/common/Topbar/fragments/PatientBanner.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { get } from "lodash";
+import moment from "moment";
+
+import Typography from '@material-ui/core/Typography';
+
+/**
+ * This component returnts banner with Patient information
+ *
+ * @author Bogdan Shcherban <bsc@piogroup.net>
+ * @param {shape} classes
+ * @param {shape} patientInfo
+ * @constructor
+ */
+const PatientBanner = ({ classes, patientInfo }) => {
+    return (
+        <div className={classes.patientInfo}>
+            <div>
+                <div className={classes.patientRightColumn}>
```

Comment:
I advise you to use material-ui Grid, it is responsive

Reply:
I've changed this component via Grid